result/rsvp.yaml

```diff
@@ -0,0 +1,192 @@
+openapi: 3.1.0
+
+info:
+  title: RSVP-TE metric results model.
+  version: ^0.0.0
+
+components:
+  schemas:
+    Rsvp.Metrics.Request:
+      description: >-
+        The request to retrieve RSVP-TE per Router metrics/statistics.
+      type: object
+      properties:
+        router_names:
+          description: >-
+            The names of RSVP-TE Routers to return results for.
+            An empty list will return results for all RSVP-TE routers.
+          type: array
+          items:
+            type: string
+          x-constraint:
+            - "/components/schemas/Device.Rsvp/properties/name"
+        column_names:
+          description: >-
+            The list of column names that the returned result set will contain.
+            If the list is empty then all columns will be returned except for
```

Comment:
If the input list is empty then all columns will be returned except for

Reply:
Done.

Reply:
Done.